ecmascript/transforms/proposal/src/decorators/legacy.rs

```diff
@@ -773,12 +773,30 @@ impl Legacy {
                 });
             }
 
-            constructor
+            let decorate_stmts_insert_position = constructor
                 .body
                 .as_mut()
                 .unwrap()
                 .stmts
-                .extend(constructor_stmts)
+                .iter()
+                .position(move |stmt| {
+                    if let Stmt::Expr(expr_stmt) = stmt {
+                        let expr_stmt = *expr_stmt.expr.clone();
```

Comment:
`expr_stmt` is a bad name for storing `Expr`. Can you change this to `expr` instead?